### ISSUE TITLE
.proselintrc example and terminal screenshot

### ISSUE THREAD

**Issue by ndarville** [amperser/proselint#316]
Would be nice to have an explicit example section for .proselintrc, including both

1. example contents
2. the relevant terminal terminal command(s)

Doesn’t have to be a terminal screenshot, just something like

```sh
$ postlint foo.markdown
# (...) [error code]
```

**Comment by mkcode** [comment]
Having the cli dump out a template file would be even better!

**Comment by ndarville** [comment]
Speaking of, what would a `.proselintrc` look like, if I just wanted to exclude `"butterick.symbols"` and `"leonard.exclamation"`?

Tried setting them to false and just deleting them, but didn’t seem to work.

**Comment by stig** [comment]
I'm also stumped about how to get a `.proselintrc` to work. I've tried putting it in `~`, where dot-files usually go, but none of the following works:

    butterick.symbols.curly_quotes
    butterick.symbols.curly_quotes=None
    butterick.symbols.curly_quotes=false

(I'm picking on that particular one because I write in Markdown/Org and the post-process step takes care of the curliness of the quotes.)

**Comment by ndarville** [comment]
cc #356.

**Comment by cpitclaudel** [comment]
Indeed, the docks only say that `You can disable any of the checks by modifying .proselintrc`.  What's the actual syntax?

**Comment by cpitclaudel** [comment]
Here's an example that works for me. In `~/.proselintrc` I put

```json
{ "checks": { "typography.diacritical_marks": false } }
```

**Comment by proinsias** [comment]
@stig – in #491, I found a reference to the fact that only the first two keys are used currently, so change your file to:
```bash
{
  "checks": {
    "typography.symbols": false
  }
}
```
